app/app.ts

```diff
@@ -19,17 +19,30 @@ const { ipcRenderer, remote } = electron;
 const slobsVersion = remote.process.env.SLOBS_VERSION;
 
 if (remote.process.env.NODE_ENV === 'production') {
-  const bugsplat = require('bugsplat')('slobs', 'slobs-renderer', slobsVersion);
-  window.onerror = (messageOrEvent, source, lineno, colno, error) => bugsplat.post(error);
+  const bt = require('backtrace-node');
```

Comment:
This is being compiled into our webpack bundle.  It's probably fine, but for something like this it's probably best to just require it dynamically at runtime.  To do that, you will need to add it to the list of externals in our webpack config.